### ISSUE TITLE
Cross section plot

### ISSUE THREAD

**Issue by MrHoneyB** [anikfal/PostWRF#32]
Can we show wind vectors in cross-section plot with PostWrf like this. 
![Screenshot_20240118_174002_Drive](https://github.com/anikfal/PostWRF/assets/120490361/ebbb8ef5-b9f4-4438-99d4-9c1cb4e34556)

**Comment by anikfal** [comment]
No. Wind vectors are not displayed on cross-section plots. However, this capability will be added to the PostWRF's cross-section plot.
At the moment, you can display wind speed (`1st_var_name = (ua^2 + va^2)^0.5`) as contour lines on the cross-section plots.

**Comment by MrHoneyB** [comment]
When the next version of postWRF is coming ?

**Comment by anikfal** [comment]
> When the next version of postWRF is coming ?

Next March probably.
